charts/em-hrs-api/values.yaml

```diff
@@ -51,8 +51,8 @@ java:
         - name: em-app-insights-connection-string
           alias: app-insights-connection-string
   environment:
-    SPRING_DATASOURCE_URL: jdbc:postgresql://em-hrs-api-postgres-v11-db-{{ .Values.global.environment }}.postgres.database.azure.com:5432/emhrs?ssl=disable
-    SPRING_DATASOURCE_USERNAME: emhrs@em-hrs-api-postgres-v11-db-{{ .Values.global.environment }}
+    SPRING_DATASOURCE_URL: jdbc:postgresql://em-hrs-api-postgres-db-v15-{{ .Values.global.environment }}.postgres.database.azure.com:5432/emhrs?ssl=disable
```

Comment:
do not we use it from vault?

Reply:
No. We don't it form the vault. This is just us following the convention with in the programme.